flatdata-generator/flatdata/generator/grammar.py

```diff
@@ -185,6 +187,8 @@ def _combine_list(t):
     Optional(comment)
 )
 
-flatdata_grammar = Group(free_comments +
-                         OneOrMore(namespace)("namespace")
-                         )("flatdata")
+flatdata_grammar = Group(
+    Optional(comment)("doc") +
```

Comment:
Not sure about this one: Most languages do not support this... and then were does it end up?

Reply:
Ok, removed.